cmd/zqd/logger/logger.go

```diff
@@ -1,20 +1,56 @@
 package logger
 
 import (
+	"fmt"
+	"io/ioutil"
+
 	"go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
 )
 
+type LoggerType string
+
+const (
+	TypeSink      LoggerType = "sink"
```

Comment:
Nit: Sink looks like it's a synonym for file. If that's the case, I think file is clearer terminology.